recipes/python-wasmer/meta.yaml

```diff
@@ -0,0 +1,152 @@
+{% set name = "python-wasmer" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: python-wasmer-split
+
+source:
+  - url: https://github.com/wasmerio/wasmer-python/archive/refs/tags/{{ version }}.tar.gz
+    sha256: 220af5b2675480daea4ebc1cede45728ce1b270cdc141ebff08ef0870cc23cfd
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - posix  # [win]
+
+outputs:
+  - name: python-wasmer
+    version: {{ version }}
+    script: build_wasmer.sh       # [unix]
+    script: build_wasmer.bat      # [win]
+    build:
+      run_exports:
+        - python-wasmer
+    requirements:
+      build:
+      - {{ compiler('c') }}
+      - {{ compiler('cxx') }}
+      - {{ compiler('rust') }}
+      - sysroot_linux-64 2.17  # [linux64]
```

Comment:
I think this line automatically requests COS7 docker image so you don't need to modify the otehr files.

Reply:
To be safe, you could include cos7 in conda-forge.yml like this:

https://conda-forge.org/docs/maintainer/knowledge_base.html#using-centos-7

```yaml
os_version:
  linux_64: cos7
```